test/FunctionalTests/Microsoft.OData.Client.Tests/Serialization/MockHttpClientHandlerProvider.cs

```diff
@@ -0,0 +1,40 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="MockHttpClientHandlerProvider.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Net.Http;
+
+namespace Microsoft.OData.Client.Tests.Serialization
+{
+    /// <summary>
+    /// A mock implementation of <see cref="IHttpClientProvider"/>
+    /// for testing purposes.
+    /// </summary>
+    internal sealed class MockHttpClientHandlerProvider: IHttpClientHandlerProvider
+    {
+        private readonly HttpClientHandler _handler;
+
+        /// <summary>
+        /// Creates a new instance of <see cref="MockHttpClientHandlerProvider"/>.
+        /// </summary>
+        /// <param name="client">The <see cref="HttpClientHandler"/> to return.</param>
+        public MockHttpClientHandlerProvider(HttpClientHandler handler, bool requestClientDisposal = false)

```

Comment:
```suggestion
        public MockHttpClientHandlerProvider(HttpClientHandler handler)
```

I think that was probably just left over from something else? I'm not seeing it used anywhere

Reply:
Yeah, this is left over from a previous implementation. Nice catch.